README.md

```diff
@@ -647,7 +706,117 @@ jmeter:
 | endpoint |  | string | Yes |
 | thresholdMillis |  | int64 | Yes |
 
+
+### Junit
+
+The check take's pod spec as input complete its job and post the result of the generated junit result files
+```yaml
+junit:
+  - testResults: "/tmp/junit-results/"
+    description: "junit demo test"
+    displayTemplate: 'Passed: [[.passed]]; Failed: [[.failed]]; Skipped: [[.skipped]]; Error: [[.error]]'
+    spec:
+      containers:
+        - name: jes
+          image: docker.io/tarun18/junit-test-pass
+          command: ["/start.sh"]
+```
+The above sample junit test will wait for the specified container to finish its job and once completed will parse all the xml files defined in `testResults` field
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| testResults | directory where the results will be published | string | Yes |
+| description| description about the test | string | No |
+| spec | Pod specification | corev1.PodSpec | Yes |
+| displayTemplate | displayTemplate represents the output format for the results. | string | No |
+
+>Note: the only thing required in spec is the containers section with only your job container
+
+This check always display results in text based format. The `displayTemplate` defaults to `Passed: [[.passed]], Failed: [[.failed]]`
+
+The fields exposed for `displayTemplate` are:
+- .passed: number of test passed.
+- .failed: number of test failed.
+- .skipped: number of test skipped.
+- .error: number of test errored out.
+
+### Smb - Verify Folder Freshness
+
+This check connect to the specified samba server to check folder freshness. This check will:
+
+- Verify the age of most recently modified file against `minAge` and `maxAge`.
+- Verify the number of files present in the mount against `minCount`.
+```yaml
+smb:
+  - server: 192.168.1.9
+    username: samba
+    password: password
+    sharename: "Some Public Folder"
+    minAge: 10h
+    maxAge: 20h
+    searchPath: a/b/c
+    description: "Success SMB server"
+```
+
+User can define server in `\\server\e$\a\b\c` format where `server` is the host `e$` is the sharename and `a/b/c` represent the sub-dir inside mount location where the test will run to verify
```

Comment:
```suggestion
The user can define server in `\\server\e$\a\b\c` format where:

- `server` is the host 
- `e$` is the sharename 
- `a/b/c` represent the sub-dir inside mount location where the test will run to verify.
```

Reply:
Not clear on what is meant by "where the test will run to verify".

Reply:
Inside the subdir in context